src/lib/components/FileUploader/FileUploaderToolbar.js

```diff
@@ -63,7 +62,7 @@ export const FileUploaderToolbar = ({
                 ? { color: 'blue' }
                 : {})}
             >
-              {humanReadableBytes(filesSize)} out of
+              {humanReadableBytes(filesSize)} out of{' '}
```

Comment:
Maybe to avoid deleting this space we can convert it to `${humanReadableBytes(filesSize)}} out of 
  ${humanReadableBytes(quota.maxStorage)}`

Reply:
After testing your solution fits better :)